### ISSUE TITLE
[SUGGESTION] Reserve the RED color in the UI / UX for only errors, validation fail and similar things

### ISSUE THREAD

**Issue by EgillAntonsson** [defold/editor2-issues#2196]
Editors, IDE 's and developer software (like VsCode) that dedicate the usage of the color RED
for errors and failed validations are in my mind the correct way to use the color.

Remember that I'm suggesting this because I care ❤️ 
I hereby strongly suggest that the Defold editor changes it UI color RED
to something else for example BLUE (The logo is BLUE and some of the UI is blue like the loading bar in the bottom right corner)
and reservers the color RED only for errors, failed validation and similar things.

Also as a more opinionated preference:
I would not have the color RED in the code color theme,
in my VsCode color theme "Bashling Theme" there is PINK color and that in my mind is fine (but not RED) and thus the red color is reserved for signaling errors (validation).
Therefore I suggest that you provide the ability to customize the color theme in the editor,
or replace the RED color in the color theme.

Compare the screenshots of VsCode and Defold
![vscode](https://user-images.githubusercontent.com/6976127/45868762-e7fa2d80-bd86-11e8-85fb-4e67ff4ef804.png)
![defold_code](https://user-images.githubusercontent.com/6976127/45868775-f0eaff00-bd86-11e8-9e17-1b8e47e36af3.png)
![defold_gui](https://user-images.githubusercontent.com/6976127/45868781-f34d5900-bd86-11e8-8060-4c03d6d1ac78.png)

The last screenshot (Defold editor) why the RED color usage is problematic:
the Alpha field has a valid value entered, but the RED underline is signaling to my mind
that it is not valid.
Using another color than RED would give the opportunity to to use it signal when values are invalid.

**Comment by vlaaad** [comment]
I agree, I too was confused initially because defold's orange color is too close to red and makes me think that currently focused input has an error. I like warm editor's feel though! Maybe make it a bit less red and more yellow?

**Comment by EgillAntonsson** [comment]
@vlaaad less red, and more yellow or other color in the Defold logo would make it better
